5 - Post DESeq2 filtering and result annotation.Rmd

```diff
@@ -0,0 +1,100 @@
+---
+title: "5 - Post DESeq2 filtering and result annotation"
+author: "Joshua Soon"
+date: '27 August 2022'
+output: html_document
+editor_options: 
+  chunk_output_type: console
+---
+
+This additional step post DESeq2 differential expression is mainly to accomplish these few objectives:
+1. Filtering out genes with less than a CPM of 0.2
+2. Annotate the filtered results with HGCN gene symbols, Entrez IDs & gene descriptions
+3. Other filtering based on Log2FoldChange or Adjust p-value.
+4. Export CPM edgeR normalized counts after CPM > 0.2 filter for GSEA.
+
+
+
+Default dependencies here and yaml are required for config.yml parsing to obtain parameters.
+```{r echo=FALSE, warning=FALSE, message=FALSE}
+library(yaml)
+```
+
+
+
+Load the packages for this specific Rmd notebook.
+```{r}
+library(org.Hs.eg.db)
+library(tidyverse)
+library(dplyr)
+
+columns(org.Hs.eg.db)
+
+library(edgeR)
+```
+
+```{r}
+source(here::here("src", "gsea_helpers.R"))
+```
+
+
+
+Drop those genes which fail to meet the low CPM cutoff.
```

Comment:
Maybe here you can add that this is optional too? It may get confusing to the user to use pre- or post-DESEq2 cpm filtering, so perhaps in the earlier Rmd where pre-DESeq2 cpm was performed you can comment there that it is for pre-DESeq2 cpm filtering, and here it is for post-DESeq2 filtering. May have to better distinguish the 2.

Reply:
Ok, mentioned that this is optional. But the user can export both annotated DE results CSV for both `RNA_seq_DE_cpm_filtered` and `RNA_seq_DE_table_rn` genes.